src/App.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {render, fireEvent} from '@testing-library/react';
+
+import App from './App';
+
+describe('UWB ACM Website', () => {
+	describe('Sponsors Page', () => {
+		describe('Seed Sponsorships Section', () => {
+			it('should render a title for the section', () => {
```

Comment:
The test framework reads really nicely, I like. Nice job on this!

Reply:
>The test framework reads really nicely, I like. Nice job on this!

😄 Hopefully RTL will make testing the behavior of our website fairly straightforward, especially when it comes to the form functionality.